src/main/java/sirius/kernel/commons/Amount.java

```diff
@@ -261,6 +262,41 @@ public Amount computeIfNull(Supplier<Amount> supplier) {
         return this;
     }
 
+    /**
+     * Invokes the given consumer with the internal value is not empty.
+     *
+     * @param consumer the consumer to execute
+     */
+    public void ifPresent(Consumer<Amount> consumer) {
+        if (isFilled()) {
+            consumer.accept(Amount.of(getAmount()));
+        }
+    }
+
+    /**
+     * Executes the given runnable if the internal value is empty.
+     *
+     * @param runnable the runnable to execute
+     */
+    public void ifEmpty(Runnable runnable) {
+        if (isEmpty()) {
+            runnable.run();
+        }
+    }
+
+    /**
+     * Returns the provided alternative {@link Amount} if the internal value is empty.
+     *
+     * @param amount the alternative Amount to return
+     * @return the original or alternative amount
+     */
+    public Amount orElseGet(Amount amount) {
```

Comment:
Rename to orElse or change Parameter to Supplier

Reply:
Renamed to orElse.
orElseGet already exists in form of computeIfNull.
Can create a new method (to keep the naming standards) and mark the other as deprecated or just leave as is.